### PR TITLE
release4.61.0

### PR DESCRIPTION
**Pull request by xinlian12**
azure-cosmos release 4.61.0, azure-cosmos-spark-* release 4.31.1

### CODE REVIEW

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM - Thanks

**Review by FabianMeiswinkel — CHANGES_REQUESTED** [review]
We need to include one more PR and make it 4.32  in Spark

File: sdk/cosmos/azure-cosmos-spark_3-1_2-12/CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,6 @@
 ## Release History
 
-### 4.32.0-beta.1 (Unreleased)
```
[the remark this answers is not in the document]
Comment (xinlian12) [reply sdk/cosmos/azure-cosmos-spark_3-1_2-12/CHANGELOG.md:3]: updated

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM now

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM now

**Pull request merged**